packages/runtime/container-runtime/src/index.ts

```diff
@@ -34,8 +34,15 @@ export {
 	IPendingMessage,
 	IPendingState,
 } from "./pendingStateManager";
-export { Summarizer } from "./summarizer";
 export {
+	IAckedSummary,
+	ISummarizer,
+	ISummarizeResults,
+	ISummaryCancellationToken,
+	neverCancelledSummaryToken,
+	Summarizer,
```

Comment:
The other day I was looking at whether we really need to export `Summarizer` - maybe not in this change since it's technically breaking, but can/should this be hidden?  Looks like only one end-to-end test uses it, but maybe that can be restructured or moved somehow (or maybe just have the test reach into the internals of the package)?

Reply:
It is also used in application code tests to create summarizer for summary related tests.